src/libraries/System.Net.Quic/src/System/Net/Quic/QuicStream.cs

```diff
@@ -390,6 +390,12 @@ public ValueTask WriteAsync(ReadOnlyMemory<byte> buffer, bool completeWrites, Ca
     /// <param name="errorCode">The error code with which to abort the stream, this value is application protocol (layer above QUIC) dependent.</param>
     public void Abort(QuicAbortDirection abortDirection, long errorCode)
     {
+
+        if (NetEventSource.Log.IsEnabled())
```

Comment:
This should get moved before the actual msquic call, since we might decide not to abort either side based on its state. So the log message may be misleading.

Reply:
updated.